src/bundle/Resources/public/js/scripts/admin.search.autocomplete.js

```
@@ -0,0 +1,141 @@
+    const clearBtn = globalSearch.querySelector(' .ibexa-input-text-wrapper__action-btn--clear');
+    const autocompleteNode = globalSearch.querySelector('.ibexa-global-search__autocomplete');
+    const autocompleteListNode = globalSearch.querySelector('.ibexa-global-search__autocomplete-list');
+    let controller;
+    const showResults = (searchText, results) => {
+        const { renderers } = ibexa.autocomplete;
+        const fragment = doc.createDocumentFragment();
+
+        results.forEach((result) => {
+            const container = doc.createElement('ul');
+            const renderer = renderers[result.type];
+
+            if (!renderer) {
+                return;
+            }
+
+            const renderedTemplate = renderer(result, searchText);
+
+            container.insertAdjacentHTML('beforeend', renderedTemplate);
+
+            const listItemNode = container.querySelector('li');
+
+            fragment.append(listItemNode);
+        });
+
+        addClickOutsideEventListener();
+        addKeyboardEventListener();
+
+        autocompleteListNode.innerHTML = '';
+        autocompleteListNode.append(fragment);
+
+        window.ibexa.helpers.ellipsis.middle.parse(autocompleteListNode);
+
+        autocompleteNode.classList.remove('ibexa-global-search__autocomplete--hidden');
+        autocompleteNode.classList.toggle('ibexa-global-search__autocomplete--results-empty', results.length === 0);
+    };
+    const getAutocompleteList = (searchText) => {
+        const url = Routing.generate('ibexa.search.suggestion', { query: searchText, limit: resultLimit });
+        const request = new Request(url, {
+            mode: 'same-origin',
+            credentials: 'same-origin',
+        });
+        controller = new AbortController();
+        const { signal } = controller;
+
+        fetch(request, { signal })
+            .then(getJsonFromResponse)
+            .then(showResults.bind(this, searchText))
+            .catch(() => {});
```

why empty catch here? Looks fishy